bin/coreos-install

```diff
@@ -26,6 +26,8 @@ fi
 : ${VERSION_ID:=current}
 CHANNEL_ID=${GROUP:-stable}
 
+BOARD="amd64-usr"
```

Comment:
@crawford Does it make sense to have a default of `amd64-usr` for this user-facing command?

I wouldn't say it's mandatory to accept this PR, but I'd prefer to see something like sourcing `/usr/share/coreos/release` and using its `$COREOS_RELEASE_BOARD` here instead (and maybe default to `amd64-usr` if that fails).  Otherwise, users running the command on ARM systems would have to be aware of this detail or they'd get an incompatible installation.